code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -471,6 +471,8 @@
 	new /obj/item/radio/headset/chameleon(src)
 	new /obj/item/stamp/chameleon(src)
 	new /obj/item/pda/chameleon(src)
+	new /obj/item/razor/straightrazor(src)
```

Comment:
please make this a normal razor and include the handmirror, as both are necessary for self haircuts

Reply:
Fixed